rlclientlib/live_model_impl.cc

```diff
@@ -69,13 +71,15 @@ namespace reinforcement_learning {
       RETURN_IF_FAIL(explore_exploit(event_id, context, response, status));
     }
     response.set_event_id(event_id);
-    RETURN_IF_FAIL(_ranking_logger->log(event_id, context, flags, response, status));
+    RETURN_IF_FAIL(_ranking_logger->log(event_id, context, flags, response, status, _decision_mode));
 
     // Check watchdog for any background errors. Do this at the end of function so that the work is still done.
     if (_watchdog.has_background_error_been_reported()) {
       RETURN_ERROR_LS(_trace_logger.get(), status, unhandled_background_error_occurred);
     }
 
+    RETURN_IF_FAIL(post_process_rank(response, _decision_mode));
```

Comment:
This should be moved before _watchdog.  _watchdog errors are the lowest priority errors.

Reply:
Fixed.

Reply:
Fixed.

---
In reply to: [385751901](https://github.com/VowpalWabbit/reinforcement_learning/pull/129#discussion_r385751901) [](ancestors = 385751901)